### PR TITLE
[improvement](build) Compile enable WITH_MYSQL

### PR DESCRIPTION
**Pull request by zxealous**
## Proposed changes

Issue Number: close #xxx

complie with mysql when run be ut, avoid compile failed with mysql

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zxealous** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]
LGTM

File: run-be-ut.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+#!/usr/bin/env bash
```
Comment (HappenLee) [inline run-be-ut.sh:2]: why add the line ?

Comment (zxealous) [reply run-be-ut.sh:2]: Misoperation, i will resubmit

**Comment by zxealous** [comment]
run buildall

File: run-be-ut.sh (hunk 1 above)
Comment (zxealous) [reply run-be-ut.sh]: done，plz help me review it

**Review by luozenglin — APPROVED** [review]
LGTM

**Review by xy720 — APPROVED** [review]

**Review by adonis0147 — APPROVED** [review]
LGTM

**Review by chenlinzhong — APPROVED** [review]
LGTM

**Comment by zxealous** [comment]
run external

**Pull request merged**